src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,15 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+        Ball randomBall1 = ball1.getRandomBall();
+        Ball randomBall2 = ball2.getRandomBall();
+        Ball randomBall3 = ball3.getRandomBall();
+        System.out.println(randomBall1.toString());
+        System.out.println(randomBall2.toString());
+        System.out.println(randomBall3.toString());
+
     }
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#please-dont-add-redundant-empty-lines-to-your-code)
Also, it would be better to use a cycle for repeatable code